tracer/src/Datadog.Trace/LifetimeManager.cs

```diff
@@ -60,7 +60,7 @@ public static LifetimeManager Instance
             }
         }
 
-        public TimeSpan TaskTimeout { get; set; } = TimeSpan.FromSeconds(30);
+        public TimeSpan TaskTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
Why are we extending this to a full minute? Are we ok with delaying shutdown that long? I think after some amount of time we should give up instead of just waiting longer. Maybe add a short comment explaining why we need 60 seconds.

(We may have to make this configurable in the future so that in serverless we can default to something _very_ short. Extending the time a AWS Lambda function's execution time has billing implications for users $$$.)

Reply:
good to know! I reverted to 30 seconds. And after running the CI in debug mode and adding several logs I realized that it's not runtime metrics timeout on .net 3.1 on windows at shutdown but it's [timing out before](https://github.com/DataDog/dd-trace-dotnet/blob/06ef37a42f32ed3862d5b91f1e2ba5751c902ff2/tracer/src/Datadog.Trace/TracerManager.cs#L734) as the other log messages after that aren't printed, so I'll revert for this PR